PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+    try:
+        Config.read('config.ini')
+        Config.set('Uzytkownik', 'test', str(wlaczniktest.get()))
+        x = open('config.ini', 'w')
+        Config.write(x)
+    except:
+        pass
+
+glownemenu = Menu(menubar, tearoff=0)
+
+oddzialmenu = Menu(menubar,tearoff=0)
+# oddzialmenu.add_radiobutton(label='02 - Kalisz', variable=ktoryoddzial, value='02', command=setnumber)
+oddzialmenu.add_radiobutton(label='04 - Katowice', variable=ktoryoddzial, value='04', command=setnumber)
+oddzialmenu.add_radiobutton(label='05 - Gdańsk', variable=ktoryoddzial, value='05', command=setnumber)
+oddzialmenu.add_radiobutton(label='07 - Poznań', variable=ktoryoddzial, value='07', command=setnumber)
+oddzialmenu.add_radiobutton(label='15 - Toruń', variable=ktoryoddzial, value='15', command=setnumber)
+oddzialmenu.add_radiobutton(label='18 - Częstochowa', variable=ktoryoddzial, value='18', command=setnumber)
+oddzialmenu.add_radiobutton(label='20 - Olsztyn', variable=ktoryoddzial, value='20', command=setnumber)
+oddzialmenu.add_radiobutton(label='50 - Centrala', variable=ktoryoddzial, value='50', command=setnumber)
+oddzialmenu.add_radiobutton(label='61 - Produkcja OEM', variable=ktoryoddzial, value='61', command=setnumber)
+menubar.add_cascade(label="Plik", menu=glownemenu)
+glownemenu.add_cascade(label='Oddział', menu=oddzialmenu)
+glownemenu.add_separator()
+
+glownemenu.add_checkbutton(label='Podpowiedzi', variable=wlaczniktooltip, command=settooltip)
+glownemenu.add_separator()
+glownemenu.add_checkbutton(label='Domyślnie włączone testowanie ciśń.', variable=wlaczniktest, command=setdefaulttest)
+
+
+glownemenu.add_separator()
+glownemenu.add_command(label="Wyjście", command=root.quit)
+
+root.config(menu=menubar)
+
+listawezy = []
+listaparametrow = ['0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0']
+
+listaklientow = []
+
+def ladujklientow():
+    try:
+        wbX = xlrd.open_workbook(
+            '//Srv-dokumenty/Swiadectwomat/bin/klient.xls',
+            formatting_info=True)
+    except FileNotFoundError:
+        wbX = xlrd.open_workbook('bin/klient.xls', formatting_info=True)
+        messagebox.showwarning('Twój INTERNET mówi NIE!',
+                               'Nie można nawiązać połączenia ze spisem klientów na serwerze sieciowym!\n\nZaładowano listę klientów z zapasowego pliku lokalnego.\n\nUWAGA - funkcja dodania nowego klienta do listy jest WYŁĄCZONA!')
+    global wsX
+    wsX = wbX.sheet_by_index(0)
+    listaklientow.clear()
+    for row in wsX.col(0):
+        key = b'Uf-FryHfa-JXDToRDClFsSrv2h56kzEUCK5imvQngbo='
+        f = Fernet(key)
+        row_value = row.value
+        kodowanie = row_value.encode()
+        odkryte = f.decrypt(kodowanie)
+        odszyfrowane = odkryte.decode('utf-8')
+        listaklientow.append(odszyfrowane)
+
+ladujklientow()
+
+def dodajklienta():
+    try:
+        wbnX = xlrd.open_workbook(
+            '//Srv-dokumenty/Swiadectwomat/bin/klient.xls',
```

Comment:
Stałe wartości dobrze wydzielić gdzieś na początek modułu/klasy z nich korzystających np. `CLIENT_SHEET = "//Srv-dokumenty/Swiadectwomat/bin/klient.xls"'. Jeśli będziesz zmienał ścieżkę to w jednym miejscu, a nie w n miejsc.